### PR TITLE
changed the cmdlet

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9057

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md [hunk 1]
```diff
@@ -40,10 +40,18 @@ Learn how to evaluate attack surface reduction rules by enabling audit mode to t
 
 Enable attack surface reduction rules in audit mode to view a record of apps that would have been blocked if the feature was fully enabled. Test how the feature will work in your organization to ensure it doesn't affect your line-of-business apps. You can also get an idea of how often the rules will fire during normal use.
 
-To enable all attack surface reduction rules in audit mode, use the following PowerShell cmdlet:
+To enable attack surface reduction rule in audit mode, use the following PowerShell cmdlet:
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md:43]:
```suggestion
To enable an attack surface reduction rule in audit mode, use the following PowerShell cmdlet:
```

**Review by adirdidi — CHANGES_REQUESTED** [review]

File: windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md [hunk 2]
```diff
@@ -40,10 +40,18 @@ Learn how to evaluate attack surface reduction rules by enabling audit mode to t
...
 
 ```PowerShell
-Set-MpPreference -AttackSurfaceReductionRules_Actions AuditMode
+Add-MpPreference -AttackSurfaceReductionRules_Ids <rule ID> -AttackSurfaceReductionRules_Actions AuditMode
+```
+
+Where `<rule ID>` is a [GUID value of ASR rule](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/attack-surface-reduction#attack-surface-reduction-rules).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md:49]:
```suggestion
Where `<rule ID>` is a [GUID value of the attack surface reduction rule](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/attack-surface-reduction#attack-surface-reduction-rules).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md (hunk 2 above)
Comment (adirdidi) [reply windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md:49]: Please use the short link I provided instead: attack-surface-reduction.md#attack-surface-reduction-rules

Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md:49]: Combined suggestion, based on all 4 comments above:
```suggestion
Where `<rule ID>` is a [GUID value of the attack surface reduction rule](attack-surface-reduction.md#attack-surface-reduction-rules).
```
Using the local file link becomes very useful when the directory `/microsoft-defender-atp/` gets renamed to `/microsoft-defender-for-endpoint/` in the future.

File: windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ ms.technology: mde
 
 **Applies to:**
 
-* [Microsoft Defender for Endpoint](https://go.microsoft.com/fwlink/?linkid=2154037)
+* [Microsoft Defender for Endpoint](https://go.microsoft.com/fwlink/p/?linkid=2146631)
```
[the remark this answers is not in the document]
Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md:25]:
```suggestion
* [Microsoft Defender for Endpoint](https://go.microsoft.com/fwlink/?linkid=2146631)
```

[the remark this answers is not in the document]
Comment (levinec) [reply windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md:25]: This fwlink should not be changed. 2154037 is the correct location

[the remark this answers is not in the document]
Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/evaluate-attack-surface-reduction.md]: Thanks, I didn't notice the number difference. Good job.

**Pull request merged**